consensus/build.rs

```diff
@@ -1,8 +1,9 @@
 use std::io::Result;
+
 fn main() -> Result<()> {
     // Proto compilation rules for `simplex` dialect
     let mut config = prost_build::Config::new();
-    config.bytes([
+    config.bytes(&[
```

Comment:
While this isn't a fix per se (it works fine), their examples do definitely use `&` and find it worth mirroring their style: https://docs.rs/prost-build/latest/prost_build/struct.Config.html#method.bytes

Would you mind making this change to all `build.rs` files (I'd like to do all at once).